module1/tasks.js

```diff
@@ -0,0 +1,189 @@
+// ==== Module#1 Basics ====
+mocha.setup('bdd');
+const expect = chai.expect;
+
+// ==== Change the capitalization of all letters in a string ====
+const changeCase = (str) => { 
+  let result = '';
+  for (let i = 0; i < str.length; i++) {
+    const char = str.charAt(i);
+    if(isUpperCase(char)) {
+      result += char.toLowerCase();
+    } else {
+      result += char.toUpperCase();
+    }
+  }
+  return result;
+};
+
+function isUpperCase(char) {
+  return char == char.toUpperCase();
+}
+function convertChar(c) {
+  return isUpperCase(c) ? c.toLowerCase() : c.toUpperCase();
+}
+
+const changeCase2 = (str) => {
+  return [...str]  //transform string to array of chars
```

Comment:
[OPTIONAL]
I see you use `[...str]`. I would say that it's a good fit for arrays when you want to create a new array from the existing one. For other data types in my opinion it's better to use `Array.from`